### PR TITLE
fix(requests): update request images with slack avatar

### PR DESCRIPTION
**Pull request by PatrickDohn**
#### Summary

-  Remove old request images and replace them with Slack Avatar.

#### Checklist

_Double check list and complete these actions before merge_

- [x] PR title is in format `TYPE(scope): description`
- [x] Link PR to Issue (if any issues are related)

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -662,9 +665,16 @@ app.action(PTO_REQUEST_APPROVE, async (response) => {
...
+  const {
+    messageBack,
+    messageRequestChannel,
+    updateRequestMessage,
+    updateMessage,
+    name,
+    messageSupportChannel,
+  } = appUser;
   const values = JSON.parse(action.value);
+  values.image = appUser.avatar;
```
Comment (davidholyko) [inline src/app.js:677]:
```
const values = {
  ...JSON.parse(action.value),
  image: appUser.avatar,
}
```

File: src/constants.js [hunk 2]
```diff
@@ -27,11 +27,6 @@ const RESOURCES = {
     // C026N9ZRR8U is the real id for hrbot-support channel
     // dev hrbots should not write to this channel instead use there own test channel.
     SUPPORT_CHANNEL_ID: isDevEnvironment ? process.env.SUPPORT_CHANNEL_ID : 'C026N9ZRR8U',
-    PTO_IMAGES: [
```
Comment (davidholyko) [inline src/constants.js:30]: 👍

File: src/utils/slackBlockKitUtil.js [hunk 3]
```diff
@@ -590,10 +587,9 @@ class SlackBlockKitUtil {
   }
 
   buildUserPendingRequestBlock(values, { status } = { status: PENDING }) {
-    const { description, eventType, eventSubType, startDate, endDate = '' } = values;
+    const { description, eventType, eventSubType, startDate, endDate = '', image } = values;
     const dates = templateStartEndDates({ startDate, endDate });
-    const imageURL = _.sample(PTO_IMAGES);
-    const imageAltText = 'Vacation animal';
+    // const imageURL = _.sample(PTO_IMAGES);
```
Comment (davidholyko) [inline src/utils/slackBlockKitUtil.js:592]: rm

File: src/app.js [hunk 4]
```diff
@@ -765,6 +786,7 @@ app.action(PTO_REQUEST_DELETE, async (response) => {
   const { appUser } = context;
   const { messageBack, messageRequestChannel, name, updateMessage } = appUser;
   const values = JSON.parse(action.value);
+  values.image = appUser.avatar;
```
Comment (davidholyko) [inline src/app.js:789]: https://github.com/f1v/hrbot/pull/248/files#r673475328

File: src/app.js [hunk 5]
```diff
@@ -738,6 +758,7 @@ app.action(PTO_REQUEST_DENY, async (response) => {
...
```
Comment (davidholyko) [inline src/app.js:761]: https://github.com/f1v/hrbot/pull/248/files#r673475328

File: src/app.js [hunk 6]
```diff
@@ -702,8 +712,18 @@ app.action(PTO_REQUEST_APPROVE, async (response) => {
       channel: values.userMessage.channel,
     });
 
+    await updateRequestMessage({
```
Comment (davidholyko) [inline src/app.js:715]: this update should also be for DENIED

File: src/app.js (hunk 5 above)
Comment (PatrickDohn) [reply src/app.js]: Fixed

File: src/app.js (hunk 4 above)
Comment (PatrickDohn) [reply src/app.js]: Fixed

File: src/utils/slackBlockKitUtil.js (hunk 3 above)
Comment (PatrickDohn) [reply src/utils/slackBlockKitUtil.js]: Fixed

File: src/constants.js (hunk 2 above)
Comment (PatrickDohn) [reply src/constants.js:30]: 👍

File: src/app.js (hunk 6 above)
Comment (PatrickDohn) [reply src/app.js:717]: fixed

File: src/utils/slackBlockKitUtil.js [hunk 7]
```diff
@@ -834,6 +856,14 @@ class SlackBlockKitUtil {
           },
         ],
       },
+      status === APPROVED ||
```
Comment (davidholyko) [inline src/utils/slackBlockKitUtil.js:859]: did prettier auto format this?

i thought it would be 
`(status === APPROVED || status === DENIED) || { ... }`

Comment (PatrickDohn) [reply src/utils/slackBlockKitUtil.js:859]: Yes let me change it to that cause I thought it looked weird to

**Review by davidholyko — APPROVED** [review]
LGTM

**Pull request merged**